scaaml_intro/train.py

```diff
@@ -77,17 +79,23 @@ def train_model(config):
             # multi gpu
             strategy = tf.distribute.MirroredStrategy()
             with strategy.scope():
-                model = get_model(input_shape, attack_point, config)
-
-                # model recording setup
                 stub = get_model_stub(attack_point, attack_byte, config)
+                file_path = f"models/{stub}.keras"
+                    
+                # model recording setup
                 cb = [
                     ModelCheckpoint(monitor="val_loss",
-                                    filepath=f"models/{stub}",
+                                    filepath=file_path,
                                     save_best_only=True),
                     TensorBoard(log_dir="logs/" + stub, update_freq="batch")
                 ]
-
+                if os.path.exists(file_path):
```

Comment:
Is this check needed? It seems more intuitive to "start training again" than to "train a certain number of epochs more".

Reply:
It's indeed used for "start training again" as an individual I have no computational resources to continue 75 epochs in single go this will be helpful.

Reply:
yes I was talking about ASCAD dataset